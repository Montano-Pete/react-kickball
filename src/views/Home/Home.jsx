import { Link, BrowserRouter as Router } from 'react-router-dom';

function Home() {
  return (
    <div>
      <Router>
        <h1>Kickball!!!</h1>
        <Link to="/teams">Go to teams</Link>{' '}
        <Link to="/players">Go to players</Link>
      </Router>
    </div>
  );
}

export default Home;
