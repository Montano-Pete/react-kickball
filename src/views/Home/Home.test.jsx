import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

it('should render the homepage', () => {
  const { container } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
