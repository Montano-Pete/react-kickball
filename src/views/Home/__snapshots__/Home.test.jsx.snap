// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the homepage 1`] = `
<div>
  <div>
    <h1>
      Kickball!!!
    </h1>
    <a
      href="/teams"
    >
      Go to teams
    </a>
     
    <a
      href="/players"
    >
      Go to players
    </a>
  </div>
</div>
`;
